src/acquisition/covidcast/csv_to_database.py

```diff
@@ -28,16 +28,21 @@ def get_argument_parser():
   parser.add_argument(
     '--log_file',
     help="filename for log output (defaults to stdout)")
+  parser.add_argument(
+    '--indicator_name',
+    nargs='?',
```

Comment:
whats the purpose of this option?  do we need it?

Reply:
It's to run acquisition on one specific indicator. We do need it.

A bit of context: the final goal is to run acquisition immediately after each specific indicator run on Cronicle, so the database gets csv files immediately when new data is available. 

Right now on Automation, acquisition runs at a specific time, on any indicators with new csv available at the time. Since acquisition process checks all indicator directories under data_dir/receiving, there's been some issue with acquisition job trying to access a indicator dir that's being written to by an indicator run at the same time on Cronicle, causing failure to both processes. This change allows the option to limit such checks to one specific indicator_dir under data_dir/receiving only.  

This fix hopefully will help separate these processes as much as possible so they don't step on each other's foot.  

I have tested both the [optional arg](https://github.com/cmu-delphi/delphi-epidata/pull/1108/commits/ccad57f167607bbe4fbe760862ca1edee03eb61a) and [lacking thereof](https://github.com/cmu-delphi/delphi-epidata/pull/1108/commits/ab913234dd1370797eb27acec538ba01b213db35). test_csv_uploading.py pass in both cases, meaning if the argument is there, csv_uploading works, and if the argument is not there, csv_uploading also works.

Reply:
whoops!  that comment was supposed to be attached to line 33 -- i was asking about the `nargs` argument to the `add_argument()` call.  🤦

Reply:
specifying `default` does that by itself, making `nargs` unnecessary.  i think you should take it out to avoid confusion but it doesnt break anything to leave it there.  ¯\\\_(ツ)_/¯